container_deployment_syntax.md

```diff
@@ -0,0 +1,422 @@
+# Container deployment configuration
+
+This document describes the structure of the VCS container configuration.  
```

Comment:
I would add: that it is runtime configuration indicates run service configuration etc. Need more details IMHO.